src/main/java/Person.java

```diff
@@ -8,8 +9,8 @@ public class Person {
     private int age;
     private Person mother;
     private Person father;
-    private List<Person> sibling;
-    private List<Person> children;
+    private List<Person> sibling = new ArrayList<>();
```

Comment:
Qua naam zou ''siblings'' (meervoud) nog beter de lading dekken. Dan is het ook consequent met ''children''. Het stond ook met ''s'' aangegeven in de variabelen lijst bij de opdracht. Copy-paste van deze requirements voorkomt dit soort typo's.